tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -2118,12 +2118,11 @@ function A(){}[$a] = $a[$c];
                 ['operators' => ['=>' => BinaryOperatorSpacesFixer::ALIGN_SINGLE_SPACE_MINIMAL]],
             ],
             'multiple exceptions catch, default config' => [
-                '<?php try {} catch (A | B $e) {}',
                 '<?php try {} catch (A   |     B $e) {}',
             ],
             'multiple exceptions catch, no space config' => [
-                '<?php try {} catch (A|B $e) {}',
                 '<?php try {} catch (A   |     B $e) {}',
```

Comment:
Maybe we could add tests with union types in properties, arguments and return types?